.eslintrc.js

```diff
@@ -90,8 +90,5 @@
 				},
 			],
 		},
-		settings: {
-			"import/resolver": "webpack",
```

Comment:
can you explalin why we don't need that and how you noticed please

Reply:
Our way of importing JS files in the project is very standard. So the default resolver is good enough.